### PR TITLE
Update wording

### PR DESCRIPTION
**Pull request by AlexEven**

### CODE REVIEW

**Comment by aleeks** [comment]
MASTER ?????????????!! 👿

**Comment by AlexEven** [comment]
"Roh ça vaaaaa"

File: AdminSelfUpgrade.php [hunk 1]
```diff
@@ -5185,7 +5185,7 @@ function doAjaxRequest(action, nextParams){
 			}
 			catch(e){
 				res = {status : "error", nextParams:nextParams};
-				alert("'.$this->trans('Javascript error (parseJSON) detected for action ', array(), 'Modules.Autoupgrade.Admin').'\""+action+"\".'
+				alert("'.$this->trans('Javascript error (parseJSON) detected for action %action%', array('%action%' => '+action+'), 'Modules.Autoupgrade.Admin').'\""+action+"\".'
```
Comment (Quetzacoalt91) [inline AdminSelfUpgrade.php]: `.'\""+action+"\".'` to be removed

**Comment by Quetzacoalt91** [comment]
Replaced by #78 with the new module structure

**Pull request closed without merging**